assets/css/default.css

```diff
@@ -19,7 +19,7 @@
   --highlight-color: rgba(137, 137, 222, 0.3);
   --font-family-base: "Newsreader";
   --font-size-base: 20px;
-  --content-max-width: 620px;
+  --content-max-inline-size: 620px;
```

Comment:
`--content-max-width` is now `--content-max-inline-size`. We could just keep the existing name if backwards compatibility is a concern, but the new name is otherwise more appropriate.